plugins/pa11y/readme.md

```diff
@@ -0,0 +1,36 @@
+# Description 
+ Pa11y is your automated accessibility testing pal. It runs accessibility tests on your pages via the command line or Node.js, so you can automate your testing process. Pa11y runs automatically on the `test:review` and `test:staging` hooks, using the information about the app from the state.
+
+# Pa11y documentation
+ `https://github.com/pa11y/pa11y`
+
+# Pa11y configurations added on the project
+    Pa11ySchema = {
+        - host: Domain against pa11y will run,
```

Comment:
suggestion: it would be great to show the type for each option, whether it's an array or string etc.